html/includes/api_functions.inc.php

```diff
@@ -12,12 +12,14 @@
  * the source code distribution for details.
  */
 
+use LibreNMS\Config;
+
 function authToken(\Slim\Route $route)
 {
     $app   = \Slim\Slim::getInstance();
     $token = $app->request->headers->get('X-Auth-Token');
     if (isset($token) && !empty($token)) {
-        if (!function_exists('get_user')) {
+        if (!function_exists('get_user') || Config::get('api_auth_local', true)) {
```

Comment:
Shouldn't this be `Config::get('api_auth_local', false)`, to be uniform with `$config['api_auth_local'] = false;` specified in the `defaults.inc.php` file?